docs/guides/using-render-tool.md

```diff
@@ -0,0 +1,78 @@
+# Using the Render tool
+
+ESO has a command called `render-template`. This command can be used to test templates for `PushSecret` and `ExternalSecret`.
+
+To run `render-template` simply execute `make build` or by `go build main.go -o bin/external-secrets` in the root folder of ESO.
+
+Once the build succeeds, the command can be used as such:
+```console
+./bin/external-secrets render-template --source-templated-object template-test/push-secret.yaml --source-secret-data-file template-test/secret.yaml
```

Comment:
this is not up to date due to the restructuring into the `render` directory

Reply:
Ah you're right. Thanks.